python/lsst/ts/ess/csc/config_schema.py

```diff
@@ -28,7 +28,7 @@
 $schema: http://json-schema.org/draft-07/schema#
 $id: https://github.com/lsst-ts/ts_ess/blob/main/python/lsst/ts/ess/csc/config_schema.py
 # title must end with one or more spaces followed by the schema version, which must begin with "v"
-title: ESS v6
+title: ESS v7
```

Comment:
Thanks for following the procedure for increasing the version number when adding new telemetry. As it happens, v6 is not in use yet at the summit so please revert this commit.

Reply:
Instead, can I just cancel this Pull Request?

Reply:
Yes, of course. Please cancel the PR and delete your branch.